ast/ddl.go

```diff
@@ -71,12 +71,25 @@ const (
 	DatabaseOptionCharset
 	DatabaseOptionCollate
 	DatabaseOptionEncryption
+	DatabaseOptionPlacementPrimaryRegion
+	DatabaseOptionPlacementRegions
+	DatabaseOptionPlacementFollowerCount
+	DatabaseOptionPlacementVoterCount
+	DatabaseOptionPlacementLearnerCount
+	DatabaseOptionPlacementSchedule
+	DatabaseOptionPlacementConstraints
```

Comment:
offset calculation base on the consecutive definition here. 
once another definition inserted among here, a reflecting map is necessary.

Reply:
It is intended. All options are same for db/table/partition anyway. I could assign every table/db enum one specific number like `0x3000 + iota`.